chia/wallet/cat_wallet/cat_wallet.py

```diff
@@ -83,7 +83,7 @@ def not_ephemeral_additions(sp: SpendBundle) -> List[Coin]:
     max_cost = DEFAULT_CONSTANTS.MAX_BLOCK_COST_CLVM
     for cs in sp.coin_spends:
         coins, cost = compute_additions_with_cost(cs, max_cost=max_cost)
-        max_cost -= cost
+        max_cost = uint64(max_cost - cost)
         if max_cost < 0:
             raise ValidationError(Err.BLOCK_COST_EXCEEDS_MAX, "non_ephemeral_additions() for SpendBundle")
```

Comment:
This is losing the type and message of the raised exception since `uint64()` would raise.

Reply:
Yes, good point. I'll update to use `int` until the end.

Reply:
As above, is any conversion needed?

Reply:
you're right that the operators return `int`, so it's not necessary. Do you think I should revert these?